cylc/flow/cfgspec/globalcfg.py

```diff
@@ -781,6 +781,17 @@
                 )
         with Conf('localhost', meta=Platform):
             Conf('hosts', VDR.V_STRING_LIST, ['localhost'])
+            with Conf('selection'):
```

Comment:
I don't think we should be duplicating this section.

I think the root of the error is that something is trying to reference the concrete platform name `localhost` rather than the abstract `<platform name>`.

We could make a change to cylc-sphinx-extensions to allow concrete references to default to the abstract sections.

Otherwise we can change the reference to point at the abstract config.